db/view/build_progress_virtual_reader.hh

```diff
@@ -127,8 +134,13 @@ class build_progress_virtual_reader {
                                 legacy_in_progress_row.append_cell(_legacy_generation_number_col, std::move(c));
                             }
                         });
+                        auto ck = adjust_ckey(scylla_in_progress_row.key());
+                        if (clustering_key::prefix_equal_tri_compare(underlying_schema())(ck, _previous_clustering_key) == 0) {
```

Comment:
It's simpler semantically and grammatically to do it like this:
```c++
if (ck.equal(*_schema, _previous_clustering_key)) {
```

Reply:
done